app/components/SwapDetails.tsx

```diff
@@ -6,19 +6,17 @@ type SwapDetailsProps = {
 };
 
 // TODO: Hack, GET /swaps response should format decimals like GET /orders ?
-function insertDecimals(num, dec) {
-  return (num / 10 ** dec).toFixed(dec);
-}
+// function insertDecimals(num, dec) {
+//   return (num / 10 ** dec).toFixed(dec);
+// }
```

Comment:
If not needed, can we remove it?

Reply:
The code is currently not used, but might be in the future again. It is currently causing linting problems, which is annoying. Since I don't know the code base that well I opted for commenting it instead of just removing it. I would potentially use/remove it once I (re-)implement the swap execution.